pkg/aws/validations/iam_helpers.go

```diff
@@ -4,9 +4,13 @@ import (
 	"fmt"
 
 	"github.com/aws/aws-sdk-go-v2/aws"
+
```

Comment:
- newline

nitpick, not sure if formatting the imports caused this

Reply:
our formatting seems to have caused this :(